mycroft/skills/core.py

```diff
@@ -295,11 +295,11 @@ def detach(self):
 
     def initialize(self):
         """
-        Initialization function to be implemented by all Skills.
+        Initialization function, run after fully constructed
```

Comment:
We might want to clarify this even more. Since the current use is to register handlers for the message bus and run methods requiring the messagebus (register entities with padatious, etc)

_Initialization function, will be run after skill is fully constructed and registered with the mycroft messagebus._

Reply:
Good call, I reworked the comments altogether.